### DISCUSSION TITLE
Please give me a single line to paste in an admin cmd.exe on windows to install this library

### DISCUSSION THREAD

**Discussion by shodanx2** [emgucv/emgucv#826] [Q&A]
I think it would be beneficial to new users to be able to install the software easily.

Preferably without having to install software to install the software to install the software, but if you want to string all that along with "&" characters it would still be fine.

**Comment by emgucv** [comment]
The easiest way to use Emgu CV is to install the nuget package. For widows, you can install this package:
https://www.nuget.org/packages/Emgu.CV.runtime.windows

If you are using command line, and want to install the package for your C# project, the following command will install the nuget package for you:

`dotnet add package Emgu.CV.runtime.windows`
